dbt/adapters/duckdb/plugins/delta.py

```diff
@@ -43,6 +44,110 @@ def load(self, source_config: SourceConfig):
     def default_materialization(self):
         return "view"
 
+    def store(self, target_config: TargetConfig, df=None):
```

Comment:
this part will be updated as integration with current `profiles` fsspec option after this PR finished https://github.com/delta-io/delta-rs/issues/570 right?

Reply:
`delta-rs`'s filesystem seems to will use `pyarrow`'s `fs` and that can be incorporated into this method with if-else or match clause
https://github.com/duckdb/dbt-duckdb/blob/c951148c45fe24c22fc86a60198d47498703c40c/dbt/adapters/duckdb/environments/__init__.py#L203

also we can instantiate `pyarrow.fs` in current existing pattern. but I'm not familar with this repository's convention
https://github.com/duckdb/dbt-duckdb/blob/c951148c45fe24c22fc86a60198d47498703c40c/dbt/adapters/duckdb/environments/__init__.py#L151

just leaving a comment to want to be of little help..

I've never contributed to open source, is there any way I can apply if I want to contribute?

Reply:
Hi @ZergRocks,
I am not sure if i follow here. As i understand, you ask if we support custom fs which can be used in the delta-rs package
https://delta-io.github.io/delta-rs/usage/loading-table/#custom-storage-backends? 
So it is currently not implemented but maybe it can be at least for begining in the read part which is done by my side, so if you want to do it i would be happy to support you. You can  write me in the slack dbt chanell and we can comment on that together

Reply:
The pyarrow.fs won't be implemented, it was a placeholder for a possible implementation but we are not continuing with that anymore.

Reply:
The pyarrow.fs won't be implemented, it was a placeholder for a possible implementation but we are not continuing with that anymore.